app/views/auth/register.phtml

```diff
@@ -5,8 +5,9 @@
 		<input type="hidden" name="_csrf" value="<?= FreshRSS_Auth::csrfToken() ?>" />
 
 		<div class="form-group">
-			<label for="new_user_name"><?= _t('gen.auth.username'), '<br />', _i('help'), ' ', _t('gen.auth.username.format') ?></label>
+			<label for="new_user_name"><?= _t('gen.auth.username') ?></label>
 			<input id="new_user_name" name="new_user_name" type="text" size="16" required="required" autocomplete="off" pattern="<?= FreshRSS_user_Controller::USERNAME_PATTERN ?>" autocapitalize="off" />
+			<p class="help"><?= _i('help') ?> <?= _t('gen.auth.username.format') ?></p>
```

Comment:
Is getting rid of the label a conscious decision? (Not an objection, I think it might make the most sense this way.)

Reply:
@Frenzie Yes, I thought it would make sense to be homogeneous with what we are doing in other places. That being said, I'm not 100% sure if there was a specific reason to bring the help in the label with a `<br>` (I supposed it was old code which was not updated, as it doesn't look like a very modern CSS-ish way to things).

Reply:
To be clear, by "getting rid of the label" I meant not using `<p><label for=>` on the help, nothing weird.

Reply:
Yep, it's what I understood ;)